src/components/UserList/index.tsx

```diff
@@ -388,7 +388,9 @@ const UserList: React.FC = () => {
                       />
                     </div>
                   </div>
-                  <div className="form-row">
+                  <div
+                    className={`form-row ${values.genpassword && 'opacity-50'}`}
```

Comment:
this will end up printing out the class name `false` when `values.genpassword` is false. You shouldn't use the `&&` pattern like this in anything except for rendering React components.

Reply:
Ah okay, fixed!